src/c++/perf_analyzer/client_backend/mock_client_backend.h

```diff
@@ -215,6 +217,7 @@ class MockClientBackend : public ClientBackend {
   Error StartStream(OnCompleteFn callback, bool enable_stats)
   {
     stats_->CaptureStreamStart();
+    stats_->enable_stats = enable_stats;
```

Comment:
CaptureStreamStart increments num_start_stream_calls. Any reason we can't use that instead of this new bool?

Reply:
This was an attempt to add additional verification for the part of the if statement that sets the `enable_stats` value (`(!parser_->IsDecoupled())`).

Reply:
Yes, verifying there's more than zero calls to `StartStream()` would also imply this if statement was accessed. It's all not a good place to add testing too. The functions are too big.